R/vdiagram-render.R

```diff
@@ -181,20 +135,20 @@ addRatingShifts <- function(vplot, vdiagramData, styles) {
 vdiagramTable <- function(reportObject){
   ratingShifts <- fetchRatingShifts(reportObject)
   
-  shiftPoints <- ratingShifts$shiftPoints
+  shiftPoints <- ratingShifts[["shiftPoints"]]
```

Comment:
Yeah, I thought the same thing, but didn't think of that solution. Good idea, will implement the fetcher and the actual rendering bit.